README.md

```diff
@@ -149,7 +152,7 @@ workflow:
   file: Snakefile
 outputs:
   files:
-    - histograms_merged.root
+    - histograms.root
```

Comment:
Note that this was fixed in the meantime in latest `master` branch. I'll amend this when merging...